release-notes/3.0/api-diff/.Net/standalone-packages/3.0.0-standalone-packages_System.IO.Pipes.md

```diff
@@ -0,0 +1,109 @@
+# System.IO.Pipes
+
+``` diff
+-namespace System.IO.Pipes {
```

Comment:
This shouldn't be a diff, we aren't shipping this (Pipes.Acls) so we can't be removing anything.  Folks can still use old package and get same API